src/components/formSections/PersonalInfoSection.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { Grid } from '@material-ui/core';
+
+import InputField from '../InputField';
+import { MajorDropdownField, YearDropdownField } from '../dropdowns';
+
+const getPersonalInfoSection = params => {
+  let readOnly;
+  if (params === undefined) {
```

Comment:
ah didn't know that - I can change that but tbh all I really want is typescript with optional params

Reply:
Oh okay. I mean that's fine too, but like in JS at least, the most common way to check if something is null or undefined is just do `variable == null` or `variable == undefined` because apparently null == undefined in JS lolol. Very jank but yeah, what you have is fine, just wanted to let u know. Otherwise, lgtm